src/spicedb-common/lang/dslang.ts

```diff
@@ -7,40 +7,39 @@ import {
   Resolver,
 } from '../parsers/dsl/resolution';
 import {
-  CancellationToken,
   Position,
   editor,
   languages,
-} from 'monaco-editor-core';
+} from 'monaco-editor';
+import * as monacoEditor from 'monaco-editor';
 
 export const DS_LANGUAGE_NAME = 'dsl';
 export const DS_THEME_NAME = 'dsl-theme';
 export const DS_DARK_THEME_NAME = 'dsl-theme-dark';
 export const DS_EMBED_DARK_THEME_NAME = 'dsl-theme-embed-dark';
 
-export default function registerDSLanguage(monaco: any) {
+export default function registerDSLanguage(monaco: typeof monacoEditor) {
   // Based on: https://github.com/microsoft/monaco-languages/blob/main/src/typescript/typescript.ts
   const richEditConfiguration = {
      
     wordPattern:
-      /(-?\d*\.\d\w*)|([^\`\~\!\@\#\%\^\&\*\(\)\-\=\+\[\{\]\}\\\|\;\:\'\"\,\.\<\>\/\?\s]+)/g,
+      /(-?\d*\.\d\w*)|([^`~!@#%^&*()\-=+[{\]}\\|;:'",.<>/?\s]+)/g,
 
     comments: {
       lineComment: '//',
-      blockComment: ['/*', '*/'],
+      blockComment: ['/*', '*/'] satisfies [string, string],
```

Comment:
These are typed as tuples, but typescript treats the literals as `string[]`. Using `satisfies` treats them as such without whacking the type.